docs/general/usage.rst

```diff
@@ -147,6 +148,13 @@ Supported options:
    + gs: gamma-sigma ratio: sigma0 RTC / gamma0 RTC (if ``measurement = gamma``)
    + sg: sigma-gamma ratio: gamma0 RTC / sigma0 ellipsoidal (if ``measurement = sigma``)
 
+ + wm: wind-modeled backscatter extracted from a Sentinel-1 OCN (ocean) product.
+   The variable `owiNrcsCmod` is used.
```

Comment:
Maybe a bit clearer: "wind-modeled backscatter extracted from the owiNrcsCmod variable of a Sentinel-1 OCN (ocean) product."

Reply:
is this better: [214f06f](https://github.com/SAR-ARD/S1_NRB/pull/138/commits/214f06f3903b2c78807ac5e662813d33f1f4475e)?